homeassistant/components/onvif/services.yaml

```diff
@@ -76,3 +76,33 @@ ptz:
             - "GotoPreset"
             - "RelativeMove"
             - "Stop"
+
+aux:
+  name: AUX
+  description: If your ONVIF camera supports PTZ auxiliary commands such as turning wiper or heater on, you will be able to send them.
```

Comment:
Are we still going to implement the custom services when we add the switches?

Reply:
I think it's wise to keep the service for now because of the massive breadth of the ONVIF protocol covering thousands of different camera models. The entities expose some common/expected functionality, but if someone wants to do something that only their device supports, the services will allow that. I think most future development will focus on the user-friendly entities.

Reply:
If someone wants a new feature we can add a new switch (description) or similar. I suggest we don't add the services until explicitly required.

Reply:
Ok I removed them.